From 35757b51c42f431c8a6454456c0f5a7cb40da274 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Thu, 31 Jan 2019 14:51:41 +0900
Subject: [PATCH 4/5] docs: update status command of nfv

This patch is to update status message of spp_nfv to add lcore info.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 docs/guides/commands/secondary/spp_nfv.rst | 1 +
 1 file changed, 1 insertion(+)

diff --git a/docs/guides/commands/secondary/spp_nfv.rst b/docs/guides/commands/secondary/spp_nfv.rst
index 5cb2d36..4057c25 100644
--- a/docs/guides/commands/secondary/spp_nfv.rst
+++ b/docs/guides/commands/secondary/spp_nfv.rst
@@ -49,6 +49,7 @@ source if it is not patched.
 
     spp > nfv 1; status
     - status: idling
+    - lcores: [1, 2]
     - ports:
       - phy:0 -> ring:0
       - phy:1
-- 
2.7.4

